### PR TITLE
QueryDirection docs

### PR DESCRIPTION
**Pull request by angrykoala**
This PR contains the documentation changes for queryDirection #888 according to [rfc-004](https://github.com/neo4j/graphql/blob/dev/docs/rfcs/rfc-004-undirected-relationship-queries.md#querydirection-option).

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-889.surge.sh

File: docs/modules/ROOT/pages/queries.adoc [hunk 1]
```diff
@@ -124,6 +124,8 @@ query Query {
 
 Note that _undirected relationships_ are only supported in queries.
 
+The xref:type-definitions/relationships.adoc#_querydirection[type definitions] for a relationship may define
+a different behaviour, the option `directed` may not be available in some cases.
```
Comment (darrellwarde) [inline docs/modules/ROOT/pages/queries.adoc:128]:
```suggestion
a different behaviour, so the `directed` option may not be available in some cases.
```

File: docs/modules/ROOT/pages/type-definitions/relationships.adoc [hunk 2]
```diff
@@ -240,21 +260,3 @@ query {
...
-    name: String!
-    posts: [Post!]! @relationship(type: "HAS_POST", direction: OUT)
-}
-
-type Post {
-    name: String!
-}
-----
-
-The relationship at `User.posts` is considered a "many" relationship. Relationships such as the one above should always be of type `NonNullListType` and `NonNullNamedType`, meaning both the array and the type inside of it should have a `!`.
```
Comment (darrellwarde) [inline docs/modules/ROOT/pages/type-definitions/relationships.adoc:260]: **suggestion (blocking):** Any reason this has been removed?

File: docs/modules/ROOT/pages/type-definitions/relationships.adoc [hunk 3]
```diff
@@ -82,6 +82,26 @@ interface ActedIn @relationshipProperties {
 }
 ----
 
+=== QueryDirection
+All relationships have a _direction_, however, when querying it is possible to perform xref:queries.adoc#_undirected_queries[undirected queries].
+When defining a **relationship** you can define the default behaviour when performing queries over this relationship with the argument `queryDirection`:
```
Comment (darrellwarde) [inline docs/modules/ROOT/pages/type-definitions/relationships.adoc:87]: **nitpick:** A bit of unnecessary use of italics and bold here?

```suggestion
All relationships have a direction, however, when querying it is possible to perform xref:queries.adoc#_undirected_queries[undirected queries].
When defining a relationship you can define the default behaviour when performing queries over this relationship with the argument `queryDirection`:
```

**Review by darrellwarde — CHANGES_REQUESTED** [review]
Broadly looks good, just a few changes suggested.

File: docs/modules/ROOT/pages/type-definitions/relationships.adoc (hunk 2 above)
Comment (angrykoala) [reply docs/modules/ROOT/pages/type-definitions/relationships.adoc:260]: Nope, it is probably a conflict that I wrongly fixed

**Review by darrellwarde — APPROVED** [review]
LGTM!

**Pull request merged**